src/posts/en/configuring-babel-6-for-node-js.md

```diff
@@ -0,0 +1,175 @@
+
+This file here `code/error.js` throws an error after the second yield in the generator but the transpiled code doesn't quite look like this.
+```js
+function* errorFulGenerator() {
+  yield "yo";
+  throw new Error("source maps are awesome");
+  return "";
+}
+
+var errorGen = errorFulGenerator();
+errorGen.next();
+errorGen.next();
+```
+
+We use this command to generate **source maps** along with the **transpiled** code *notice the `--source-maps flag`*
+```bash
+babel code/ -d build/ --source-maps
+```
+
+Now when we encounter the error we get useful debugging such as this
+```js
+errorGen.next()
+         ^
+
+Error: source maps are awesome
+    at errorFulGenerator (/home/programreneur/Programming/githubRepos/babeljs-short-tutorial/code/error.js:3:9)
+    at next (native)
+    at Object.<anonymous> (/home/programreneur/Programming/githubRepos/babeljs-short-tutorial/code/error.js:10:10)
+    at Module._compile (module.js:425:26)
+    at Object.Module._extensions..js (module.js:432:10)
+    at Module.load (module.js:356:32)
+    at Function.Module._load (module.js:313:12)
+    at Function.Module.runMain (module.js:457:10)
+    at startup (node.js:138:18)
+    at node.js:974:3
+```
+So this is how you'll use source maps
+
+#### Setting up npm command
+In order to simplify the build process even more. You can update your `package.json` file to include a build script for Babel. In `package.json` script object you can add a build script such as the one below
+```json
+"scripts": {
+  "build": "babel -w code/ -d build -s"
+}
+```
+Now, we can run
+```bash
+npm run build
+```
+and get all the ES2015/ES2016 goodness instantly today. :)
+
+
+#### Learn more about Babel
+This is a basic tutorial on Babel but the Babel world just starts here. It's surrounded by a wonderful community and is used by big names in IT world. Babel has support for all the major build tools too such as [Grunt](https://www.npmjs.com/package/grunt-babel) and [gulp](https://npmjs.org/package/gulp-babel/). You can check them all out here on [Babel Website](https://babeljs.io/docs/setup/)
+
+These are some of the resources that can even up your game further in the Babel world
+- [Learn ES2015 and Babel using this detailed tutorial](http://ccoenraets.github.io/es6-tutorial/index.html)
+- [Read the Babel docs on setting up Babel (They're helpful)](https://babeljs.io/docs/setup/)
+
+
+##### Source code and Contributions and Thank yous
+Source code for this tutorial is available in this [repo](https://github.com/abdulhannanali/babel-configuration-tutorial).
+
+If you find some typo or would like to make some update. Please do so using the power of issues and PR in our [Github Repo](https://github.com/abdulhannanali/babel-configuration-tutorial).
```

Comment:
full stop -> comma
You can add a link to the [JS Rocks repo](https://github.com/JSRocksHQ/jsrockshq.github.io) too if you'd like.